app/assets/javascripts/student_profile/services_list.jsx

```diff
@@ -63,7 +63,7 @@ import _ from 'lodash';
     },
 
     wasDiscontinued: function(service) {
-      return (service.discontinued_by_educator_id !== null);
+      return (service.discontinued_recorded_at !== null);
```

Comment:
The field `discontinued_at` has replaced `DiscontinuedServices`. If a service is active then its `discontinued_at` will be not `nill`.